libdnf/libdnf.pc.in

```diff
@@ -3,8 +3,9 @@ libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
 includedir=${prefix}/include
 
 Name: libdnf
-Description: Simple package manager interface and librepo
+Description: Simple package manager interface using libsolv and librepo
 Version: @LIBDNF_VERSION@
-Requires: glib-2.0, gobject-2.0, librepo, rpm
+Requires: glib-2.0, gobject-2.0, librepo, rpm, sqlite3
```

Comment:
`sqlite3` is a build time option today for the SWDB, so it'd be more correct to append this only conditionally based on the build-time options.

And even then it seems to me it's a bug that `dnf-swdb-db.h` has private underscore `_` prefixed methods, those should be in `dnf-swdb-db-private.h` or so?

Reply:
I do conditionally add it, see the the third commit. ;)

Unfortunately, the API exposure is required for DNF and PK for now.

Reply:
Ah, OK.

Reply:
Why do you say that the SWDB underscore prefixed API is required for PK? I don't think PK uses any SWDB API at all.

Reply:
Because once SWDB lands, the PK-Dnf backend needs to be reworked to use it so that we stop having stupidity with transaction history gaps and such.